backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/database/Schema.kt

```diff
@@ -35,6 +36,7 @@ object Administrators : IntIdTable() {
             (deleted eq Op.TRUE and (role eq Role.NO_RIGHTS.db_value)) or
                 (deleted eq Op.FALSE and (role neq Role.NO_RIGHTS.db_value))
         }
+        uniqueIndex(functions = listOf(email.lowerCase()))
```

Comment:
To get rid of the remaining warning as well, use a custom index name:
```suggestion
        uniqueIndex(customIndexName = "email_lower_idx", functions = listOf(email.lowerCase()))
```

Reply:
Nice ^^ for me there was no remaining warning.